test/unit/RegistryCoordinatorUnit.t.sol

```diff
@@ -853,157 +1493,285 @@ contract RegistryCoordinatorUnit is MockAVSDeployer {
             emptyAVSRegSig
         );
     }
+}
 
-    function testEjectOperatorFromCoordinator_AllQuorums_Valid() public {
-        // register operator with default stake with default quorum number
+contract RegistryCoordinatorUnitTests_UpdateOperators is RegistryCoordinatorUnitTests {
+    function test_updateOperators_revert_paused() public {
+        cheats.prank(pauser);
+        registryCoordinator.pause(2 ** PAUSED_UPDATE_OPERATOR);
+
+        address[] memory operatorsToUpdate = new address[](1);
+        operatorsToUpdate[0] = defaultOperator;
+
+        cheats.expectRevert(bytes("Pausable: index is paused"));
+        registryCoordinator.updateOperators(operatorsToUpdate);
+    }
+
+    // @notice tests the `updateOperators` function with a single registered operator as input
+    function test_updateOperators_singleOperator() public {
+        // register the default operator
+        ISignatureUtils.SignatureWithSaltAndExpiry memory emptySig;
+        uint32 registrationBlockNumber = 100;
         bytes memory quorumNumbers = new bytes(1);
         quorumNumbers[0] = bytes1(defaultQuorumNumber);
-        ISignatureUtils.SignatureWithSaltAndExpiry memory emptySig;
-
         stakeRegistry.setOperatorWeight(uint8(quorumNumbers[0]), defaultOperator, defaultStake);
-
-        cheats.prank(defaultOperator);
+        cheats.startPrank(defaultOperator);
+        cheats.roll(registrationBlockNumber);
         registryCoordinator.registerOperator(quorumNumbers, defaultSocket, pubkeyRegistrationParams, emptySig);
 
-        cheats.expectEmit(true, true, true, true, address(blsApkRegistry));
-        emit OperatorRemovedFromQuorums(defaultOperator, quorumNumbers);
+        address[] memory operatorsToUpdate = new address[](1);
+        operatorsToUpdate[0] = defaultOperator;
 
-        cheats.expectEmit(true, true, true, true, address(stakeRegistry));
-        emit OperatorStakeUpdate(defaultOperatorId, uint8(quorumNumbers[0]), 0);
+        // TODO: any additional checks here? mostly this just calls the StakeRegistry, so more appropriate for an integration test
+        registryCoordinator.updateOperators(operatorsToUpdate);
```

Comment:
Probably worth having a test that checks that if the StakeRegistry returns quorums to remove from the operator, that they're removed?

Reply:
hmm yeah, I wasn't sure if this made more sense in "integration" tests. right now these tests use a mock StakeRegistry, but I'll see if I can cook a good test up here.

Reply:
test added in https://github.com/Layr-Labs/eigenlayer-middleware/pull/121/commits/1f1a840d0b83c27e71a940f9e64ae4bb0a1863ba
I think this test worked as a minimal version, but I added a more thorough fuzzed one.